content/Services/RHOBS/rules-and-alerting.md

```diff
@@ -54,27 +76,43 @@ groups:
+```
+
+obsctl uses the credentials you saved earlier, to make an authenticated `application/yaml` `PUT` request to the `api/v1/rules/raw` endpoint of the Observatorium API, which **creates** your alerting rule.
+
+obsctl should print out the response, which, if successful, would be: `successfully updated rules file`.
+
+Besides checking this response, you can also list or **read** the configured rules for your tenant by,
 
 ```bash
-curl -X PUT --data-binary @alerting-rule.yaml --header "Content-Type: application/yaml" http://localhost:8080/api/metrics/v1/rhobs/api/v1/rules/raw
+obsctl metrics get rules.raw
 ```
 
-Besides checking if you've got a 200 response, you can also list the rules for the tenant (in this case, `rhobs`):
+This would make a `GET` request to `api/v1/rules/raw` endpoint and return the rules you configured in YAML form. This endpoint will immediately reflect any newly set rules.
+
+Note that in the response a `tenant_id` label for the particular tenant was added automatically. Since Observatorium API is tenant-aware, this extra validation step is also performed. Also, in the case of rules expressions, the `tenant_id` labels are injected into the PromQL query, which ensures that only data from a specific tenant is selected during evaluation.
+
+You can also check your rule's configuration, health, and resultant alerts by,
 
 ```bash
-curl http://localhost:8080/api/metrics/v1/rhobs/api/v1/rules/raw
+obsctl metrics get rules
 ```
 
-Note that in the response a `tenant_id` label for the particular tenant was added automatically. Since Rules/Observatorium API is tenant-aware, this extra validation step is also performed. Also, in the case of Rules expressions, the `tenant_id` labels are injected into the PromQL query, which ensures that only data from a specific tenant is selected during evaluation.
+This would make a `GET` request to `api/v1/rules` endpoint, and return rules you configured in [Prometheus HTTP API format JSON response](https://prometheus.io/docs/prometheus/latest/querying/api/#rules). You can read more about checking rule state [here](#check-the-alerting-rule-state).
+
+Note that this endpoint does not reflect newly set rules immediately and might take up to a minute to sync.
 
 ### How to update and delete an alerting rule
 
-As mentioned in the [upstream docs](https://observatorium.io/docs/design/rules-api.md/#example-request) that each time a `PUT` request is made to the `/api/v1/rules/raw` endpoint, the rules contained in the request will overwrite all the other rules for that tenant.
+As mentioned in the [upstream docs](https://observatorium.io/docs/design/rules-api.md/#example-request) that each time a `PUT` request is made to the `/api/v1/rules/raw` endpoint, the rules contained in the request will overwrite all the other rules for that tenant. Thus, each time you use `obsctl metrics set --rule.file=<file>` it will overwrite all other rules for a tenant with the new rule file.
```

Comment:
I would not be afraid to even be **bold** here :smile: 
```suggestion
As mentioned in the [upstream docs](https://observatorium.io/docs/design/rules-api.md/#example-request) that each time a `PUT` request is made to the `/api/v1/rules/raw` endpoint, the rules contained in the request will overwrite all the other rules for that tenant. **Thus, each time you use `obsctl metrics set --rule.file=<file>` it will overwrite all other rules for a tenant with the new rule file.**
```